src/main/java/chatBot/bot/Bword.java

```diff
@@ -0,0 +1,58 @@
+package chatBot.bot;
+
+import chatBot.command.Command;
+
+/**
+ * Bword is a chatbot that helps with planning tasks
+ *
+ * The constructor method takes in the location of the file to read and write data
+ * as a String, and has respective attributes of storage, tasklist and ui
+ *
+ * The run method loops continuously until a command to exit is detected.
+ *
+ * The main method initialises the program
+ */
+
+public class Bword {
+    private static final String FILELOCATION = "./data/bword.txt";
+    //public static final String HLINE = "____________________________________________________________\n";
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    private Bword(String s) {
+        this.ui = new Ui();
+        this.storage = new Storage(s);
+        try {
+            this.taskList = new TaskList(this.storage.load());
+        } catch (Exception e) {
+            ui.showLoadingError();
+            this.taskList = new TaskList();
+        }
+    }
+
+    private void run() {
+        this.ui.showWelcome();
+        boolean isExit = false;
```

Comment:
I like how you name your boolean here with prefix 'is'.

Reply:
thanks